promgen/templates/promgen/service_block_panel_notifiers.inc.html

```diff
@@ -0,0 +1,7 @@
+{% load promgen %}
+
+{% if service.notifiers.count %} 
+<div class="panel panel-default">
+  {% include "promgen/notifier_block.html" with object=service show_edit=1 %}
+</div>
+{% endif %} 
```

Comment:
Bit of a nitpick, but most every other file ends with a blank line and since GitHub is showing this as a warning, we should probably be consistent and end each file with a blank line.